WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -550,7 +550,7 @@ public void onMediaChanged(MediaStore.OnMediaChanged event) {
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
-    public void OnMediaListFetched(MediaStore.OnMediaListFetched event) {
+    public void onMediaListFetched(MediaStore.OnMediaListFetched event) {
```

Comment:
While we're at it, could we conventionalize `OnEvent` imports and update them all to fit it? IMO we should just use `OnMediaListFetched` instead of `MediaStore.OnMediaListFetched`, the relevant store/model is always contained in the naming of the `OnChanged` event.

This is currently troublesome because `OnChanged` events are non-static inner classes, and AS doesn't suggest more specific imports for those. I don't see why we can't make all `OnChanged` classes static though, just like custom `Payload`s are.